run.py

```diff
@@ -0,0 +1,26 @@
+'''
+Settings and start-script for activating the VUSualizer on the server.
+'''
+
+from src import app as application
+import sys
+
+# /data/vusualizer/VUSualizer, use location of VUSualizer on the serverside
+sys.path.insert(0, '/data/vusualizer/VUSualizer')
```

Comment:
Could this be configurable? Either via a config file or parameter parsing?
Seems to be the only difference between `run.py` and `run_local.py`. Please avoid code duplications.